mullvad-types/src/features.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashSet;
+
+use serde::{Deserialize, Serialize};
+
+/// Feature indicators are active settings that should be shown to the user to make them aware of
+/// what is affecting their connection at any given time.
+///
+/// Note that the feature indicators are not ordered.
+#[derive(Default, Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct FeatureIndicators {
+    pub active_features: HashSet<FeatureIndicator>,
+}
+
```

Comment:
Maybe this should just be a tuple struct so we avoid having to name the only field? I.e. FeatureIndicators(HashSet<FeatureIndicator>).